templates/pro/devices.html

```diff
@@ -0,0 +1,529 @@
+{% extends "templates/base.html" %}
+
+{% block title %}Ubuntu Pro for Devices{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1i_3nPkSpCQJ4jfOwN380sdfuTxaiDuDIgDfk01wVPqo/edit
+{% endblock
+meta_copydoc %}
+
+{% block meta_description %}
+  Ensure your embedded devices are fully secure and compliant with cybersecurity regulations with Ubuntu Pro for Devices. Upgrade your desktop or server images today for 10 years of security maintenance and updates.
+{% endblock meta_description %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block outer_content %}
+
+  {% block content %}
+    <section class="p-strip">
+      <div class="row">
+        <div class="col-medium-6 col-9 col-start-large-4">
+          <h1>Ubuntu Pro for Devices</h1>
+          <p>
+            Pay a one-time-fee per device and get 10 years of security maintenance. Ideal to comply with the Cyber Resilience Act and other regulations.
+          </p>
+          <a href="/pro/devices#get-in-touch" class="p-button--positive">Get in touch to learn more</a>
+        </div>
+      </div>
+      <div class="p-suru u-no-padding--top"></div>
```

Comment:
You can wrap the containing `p-strip` with `p-suru` to add the decorator to the end of the strip.